Makefile

```diff
@@ -1,4 +1,4 @@
-SHELL=/bin/bash
+SHELL=env bash
```

Comment:
This is need locally here on my machine, where isolation of packages breaking `/bin/bash`. 
I think this line is not required, but I've used the env` strategy to find where is bash.